### PR TITLE
added MPMediaLibrary permissions / getStatus

### PR DESCRIPTION
**Pull request by hippopotamus**

### CODE REVIEW

File: ReactNativePermissions.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const RNPTypes = {
 		'notification',
 		'backgroundRefresh',
 		'speechRecognition',
+        'mediaLibrary',
```
Comment (viktorsec) [inline ReactNativePermissions.js]: Improper indentation. 👼

Comment (PeterEmanuel) [reply ReactNativePermissions.js]: An upvote for this! I want to be able to download mp3 files on iOS devices. For Android I have the get permissions use case working... For iOS I ran into the 'storage' not defined error and with some Googling I got here at this pull request.
Is it also possible to make the latest version available on NPM?
Thanks!

**Comment by hippopotamus** [comment]
hey, sorry for the wait. i changed it to a hard tab.
also, fyi, this one works for iOS 9.3+

**Comment by PeterEmanuel** [comment]
A question for the pull-requester (@hippopotamus2) : I have tried to clone this library and link it locally with npm commands, in order to apply your changes, but this does not yet work for me...

Do you maybe have a publicly available repo (that is a fork of this) with this change?

I need it very much to be able to download mp3 files into iOS... For Android this works great, but not yet for iOS and the maintainer of react-native-permissions does not seem to be active anymore...

Thanks in advance, I hope you read this.

**Comment by hippopotamus** [comment]
hey peter,
https://github.com/jigsawxyz/react-native-permissions
let me know if you have any problems

**Comment by PeterEmanuel** [comment]
I will check it out, thank you very much!

**Comment by PeterEmanuel** [comment]
Thanks very much! I pointed to your fork in my package.json and after some cleaning and linking I was able to get the permission dialog working for the mediaLibrary.
Now my next Google task is to find out how I can store mp3 files in a location where the user can find and use them... react-native-fs downloads great, but only to folders in the apps sandbox... :-/

**Comment by zoontek** [comment]
This feature has been merged into master via another PR! Will be available soon

**Pull request closed without merging**